app/controllers/answers.js

```diff
@@ -1,55 +1,43 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    Answer = mongoose.model('Answer'),
-    _ = require('underscore');
+const mongoose = require('mongoose'),
+  async = require('async'),
+  Answer = mongoose.model('Answer'),
+  _ = require('underscore');
 
 
-/**
- * Find answer by id
- */
-exports.answer = function(req, res, next, id) {
-    Answer.load(id, function(err, answer) {
-        if (err) return next(err);
-        if (!answer) return next(new Error('Failed to load answer ' + id));
-        req.answer = answer;
-        next();
-    });
+exports.answer = function (req, res, next, id) {
+  Answer.load(id, (err, answer) => {
+    if (err) return next(err);
+    if (!answer) return next(new Error(`Failed to load answer ${id}`));
+    req.answer = answer;
+    next();
+  });
 };
 
-/**
- * Show an answer
- */
-exports.show = function(req, res) {
-    res.jsonp(req.answer);
+exports.show = function (req, res) {
+  res.jsonp(req.answer);
 };
 
-/**
- * List of Answers
- */
-exports.all = function(req, res) {
-    Answer.find({official:true}).select('-_id').exec(function(err, answers) {
-        if (err) {
-            res.render('error', {
-                status: 500
-            });
-        } else {
-            res.jsonp(answers);
-        }
-    });
+exports.all = function (req, res) {
```

Comment:
Your code flow is superb, how ever it would be better to use Es6 arrow function here, for better integration

Reply:
okay, Thanks. Would work on it in my next PR.